Project-Bootcamp/views/header-view.js

```diff
@@ -0,0 +1,53 @@
+var React = require ('react');
+var ReactRouter = require ('react-router');
+var Link = ReactRouter.Link;
+var Router = require ('react-router');
+var classNames = require ('classnames');
+var RouteHandler = Router.RouteHandler;
+
+var HeaderView = React.createClass ({
+
+    render: function () {
+
+        return (
+            <div {...this.getProps()}>
+                <ul className='nav navbar-nav'>
+                    <li>
+                        <Link to='Home' >Home</Link>
+                    </li>
+                    <li>
+                        <Link to='RealmStatus' >Realm Status</Link>
+                    </li>
+                    <li>
+                        <Link to='CharacterInformation' >Character Information</Link>
+                    </li>
+                    <li>
+                        <Link to='ChallengerChart' >Challenger Leaderboards</Link>
+                    </li>
+                    <li>
+                        <Link to='PvpChart' >PvP Leaderboards</Link>
+                    </li>
+                </ul>
+                <div className='row' >
+                    <RouteHandler />
```

Comment:
Your route handler shouldn't be here, it should be in an upper level (please refer to [this comment](https://github.com/alvapantanetti/startup/pull/5/files#r43503602)).